### DISCUSSION TITLE
How can I figure out the Unicode code points supported by a Nerd Font?

### DISCUSSION THREAD

**Discussion by injust** [ryanoasis/nerd-fonts#1675] [Q&A] [answered]
I tried using `otfinfo --unicode ~/Library/Fonts/SymbolsNerdFont-Regular.ttf`, but it exits 1 with `otfinfo: bad table 1 length (format 12)`.

**Comment by Finii** [comment, marked as answer]
I'm not sure what you want to do.

Here is a list of the rough ranges: https://github.com/ryanoasis/nerd-fonts/wiki/Glyph-Sets-and-Code-Points
The codepoint of individual icons can be found there: https://www.nerdfonts.com/cheat-sheet
Checking if one specific codepoint is in which installed font: `fc-list :charset=f000` (assuming you want to check `u_F000`)
Get a more or less conceise list of codepoints in the font file: `fc-query --format='%{charset}\n' ~/Library/Fonts/SymbolsNerdFont-Regular.ttf`

Never needed / even heard of `otfinfo`

_Edit: Correct cheat sheet link_

**Reply by Finii** [reply]
From `~/Library` I guess you are on Mac. I have no clue what to use there, they do not have fontconfig (which are the commands starting with `fc-`)

**Reply by injust** [reply]
fontconfig via Homebrew worked. Thanks for the answer!
